keyboards/ergodone/keymaps/vega/keymap.c

```diff
@@ -0,0 +1,581 @@
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+enum layer_names {
+    BASE,
+    GREL,
+    GREU,
+    SYMB,
+    MATH,
+    QWER,
+    FNLR
+};
+
+enum custom_keycodes {
+  PLACEHOLDER = SAFE_RANGE,
```

Comment:
What is this for? I don't think it's actually needed, you can just declare `VRSN = SAFE_RANGE` and `RGB_SLD` here.

Reply:
To be honest, I have no idea. A lot of this layout was built by trial and error and sticking blobs of code in from other boards or using config.qmk.fm , so I'm not sure what's just vestigial from that. At some point I decided, "ehh, it works" tried to clean it up to a reasonable extent, then thought I'd see if it's worth putting in the repo. I'm not entirely sure what you're suggesting here either? This?
```
 enum custom_keycodes {
  VRSN = SAFE_RANGE,
  RGB_SLD
};
```
Of note, my board doesn't even have any RGB-ness to it, but I kept a lot of the other stuff from the sources I 'stole from' in an effort to make it applicable regardless. I'll happily change whatever as long as the layout stays the same and it works.

Reply:
Ah, right. I didn't notice you had no RGB stuff here - that keycode can be removed too, if you want. And if you don't need the `VRSN` keycode either (though I see it's referenced in the keymap), you can get rid of this enum, `process_record_user()` and the version.h include at the top. That will save you a bit of flash space.

Reply:
Just removed those, compiled, and uploaded it to my keeb with this done. 👍